modules/docs/arrow-docs/docs/docs/datatypes/option/README.md

```diff
@@ -18,13 +18,25 @@ Arrow models the absence of values through the `Option` datatype similar to how
 import arrow.*
 import arrow.core.*
 
-val someValue: Option<String> = Some("I am wrapped in something")
+val message: String? = "I am wrapped in something"
+
+val someValue: Option<String> = Some(message)
 someValue
+
+// Or with the toOption extention
+val otherValue: Option<String> = message.toOption()
+otherValue
 ```
 
 ```kotlin:ank
 val emptyValue: Option<String> = None
 emptyValue
+
+// Or with the toOption extention
```

Comment:
These blocks only evaluate the last statement of each! For both emptyvalue and othervalue to be displayed these lines need to be wrapped in another ```kotlin:ank block